tcms/testruns/templates/testruns/get.html

```diff
@@ -117,6 +117,65 @@ <h2 class="card-pf-title" style="text-align: left">
+                                            <ul class="dropdown-menu">
+                                                <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                                <li><a href="#">{% trans "Priority" %}</a></li>
+                                                <li><a href="#">{% trans "Category" %}</a></li>
+                                            </ul>
+                                        </div><!-- /input-group-btn -->
+                                        <input type="text" class="form-control" id="filter" placeholder='{% trans "Filter By Name..." %}'>
+                                    </div><!-- /input-group -->
+                                </div>
+                                <div class="form-group">
+                                    <div class="dropdown btn-group">
+                                        <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
+                                        <ul class="dropdown-menu">
+                                            <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                            <li><a href="#">{% trans "Priority" %}</a></li>
+                                            <li><a href="#">Last {% trans "Category" %}</a></li>
+                                        </ul>
+                                    </div>
+                                    <button class="btn btn-link" type="button">
+                                        <span class="fa fa-sort-alpha-asc"></span>
+                                    </button>
+                                </div>
+                                <div class="form-group">
+                                    <button type="button" class="btn btn-default">{% trans "Remove case" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Update case text" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Change assignee" %}</button>
+                                    <div class="btn-group">
+                                        <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">{% trans "Change status" %} <span class="caret"></span></button>
+                                        <ul class="dropdown-menu">
+                                            {% for status in execution_statuses %}
+                                                <li><a href="#">{{ status.name }}</a></li>
```

Comment:
Remove the `#` sign b/c when you click on these menus the page jumps up & down in this case. Another alternative is to have an anchor for the card title or the card itself and point the href attribute to it. This way the toolbar will always be at the top of the page when clicking on it.

Reply:
it does not matter what the behavior now is. when this is finished this `a` element will have a js handler that will make the API calls, refresh the page/ re-render the table and return false (hence the link won't be executed)

Reply:
What matters is that this represents unfinished work which could easily be handled together with the relevant changes. If you do have a `return false` then the anchor is not honored by the browser. However this is not currently the case with the JS code on the master branch.

So one of the 3 needs to happen: 
1) Handle it now
2) Future proof the code on master for later adoption in the refactoring branch
3) Add more TODO items in the code & backlog

Which one do you prefer ?

Reply:
I prefer 3) because it's just one line of code